sqlglot/parser.py

```diff
@@ -1764,7 +1764,11 @@ def _parse_stored(self) -> exp.FileFormatProperty:
     def _parse_property_assignment(self, exp_class: t.Type[E], **kwargs: t.Any) -> E:
         self._match(TokenType.EQ)
         self._match(TokenType.ALIAS)
-        return self.expression(exp_class, this=self._parse_field(), **kwargs)
+        field = self._parse_field()
+        if isinstance(field, exp.Identifier) and not field.quoted:
+            field = exp.var(field)
```

Comment:
what triggered this?

Reply:
because we have some ddl that is like diststyle all, and all got quoted